### PR TITLE
Fix ServiceHelperExtension::conf

### PR DESCRIPTION
**Pull request by mishako**
It was using the key argument as both key and value.

### CODE REVIEW

**Comment by mishako** [comment]
The diff looks weird because I renamed `ServiceHelperExtensionTest` to `ServiceHelperExtensionUsageTest`.

**Comment by klaraward** [comment]
Curious how you discovered this and what bugs it caused?

**Comment by mishako** [comment]
@klaraward I was migrating a test to junit5, switched from `ServiceHelper` to `ServiceHelperExtension` and it started failing. It was easy to fix on my side though: by converting my int value to a string.

File: apollo-test/src/main/java/com/spotify/apollo/test/ServiceHelperExtension.java [hunk 1]
```diff
@@ -183,7 +183,7 @@ public ServiceHelperExtension conf(String key, String value) {
 
   @Override
   public ServiceHelperExtension conf(String key, Object value) {
-    serviceHelper.conf(key, key);
+    serviceHelper.conf(key, value);
```
Comment (lutovich) [inline apollo-test/src/main/java/com/spotify/apollo/test/ServiceHelperExtension.java:186]: 😳

File: apollo-test/src/test/java/com/spotify/apollo/test/ServiceHelperExtensionTest.java [hunk 2]
```diff
@@ -19,35 +19,27 @@
  */
 package com.spotify.apollo.test;
 
-import static org.junit.jupiter.engine.descriptor.JupiterEngineDescriptor.ENGINE_ID;
-import static org.junit.platform.engine.discovery.DiscoverySelectors.selectClass;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
```
Comment (lutovich) [inline apollo-test/src/test/java/com/spotify/apollo/test/ServiceHelperExtensionTest.java:23]: Maybe use `org.hamcrest.MatcherAssert#assertThat()` instead?
`org.junit.Assert#assertThat()` is from JUnit 4 and this is a JUnit 5 test

Comment (lepistone) [reply apollo-test/src/test/java/com/spotify/apollo/test/ServiceHelperExtensionTest.java:23]: Good point! I'll merge anyway. PR very welcome!

**Pull request merged**